microservices/configcheck/src/main/java/datawave/microservice/configcheck/CommandRunner.java

```diff
@@ -0,0 +1,225 @@
+                            + "  Compares two analyses, and produces a git-merge-like diff showing value differences \n"
+                            + "             between the two files.\n\n";
+        }
+        
+        return output;
+    }
+    
+    private String runRenderCommand(ApplicationArguments args) {
+        // load the content
+        String content = loadContent(getFile(args));
+        
+        // load the properties (properties, or yaml)
+        Properties properties = getProperties(args);
+        
+        return handleOutput(renderContent(content, properties));
+    }
+    
+    private String runAnalyzeCommand(ApplicationArguments args) {
+        // load the xml
+        String xmlContent = loadContent(getFile(args));
+        
+        // load the properties (properties, or yaml)
+        Properties properties = getProperties(args);
+        
+        XmlPropertyAnalyzer analyzer = new XmlPropertyAnalyzer(xmlContent, properties);
+        String output;
+        if (args.containsOption(FULL_REPORT)) {
+            output = analyzer.getReport();
+        } else {
+            output = analyzer.getKeyedValues();
+        }
+        return handleOutput(output);
+    }
+    
+    private String runCompareCommand(ApplicationArguments args) {
+        String output = null;
+        String[] files = getFiles(args);
+        if (files != null) {
+            // load the content
+            String first = loadContent(files[0]);
+            String second = loadContent(files[1]);
+            
+            AnalysisComparator comparator = new AnalysisComparator(files[0], first, files[1], second);
+            output = comparator.compareAnalyses();
+        }
+        return handleOutput(output);
+    }
+    
+    private String getConfigdir(ApplicationArguments args) {
+        String configdir = null;
+        if (args.getOptionNames().contains(CONFIGDIR)) {
+            configdir = args.getOptionValues(CONFIGDIR).get(0);
+        }
+        return configdir;
+    }
+    
+    private Properties getProperties(ApplicationArguments args) {
+        String configdir = getConfigdir(args);
+        
+        // load the properties (or yaml)
+        Properties mergedProperties = null;
+        if (args.getOptionNames().contains(PROPERTIES)) {
+            mergedProperties = loadProperties(configdir, getFileList(args, PROPERTIES));
+        } else if (args.getOptionNames().contains(YAML)) {
+            mergedProperties = loadYamlAsProperties(configdir, getFileList(args, YAML));
+        } else {
+            log.info("No properties or yaml to render");
+        }
+        
+        return mergedProperties;
+    }
+    
+    private String handleOutput(String output) {
+        String outputPath = getOutputPath(args);
+        if (outputPath != null) {
+            writeOutput(getFilePath(outputPath), output);
+            output = null;
```

Comment:
output = "SUCCESSFUL"

Reply:
im just going to output an empty string on success.  I'll output an error message if anything goes wrong.

Reply:
updated.  let me know what you think.